src/basic_grasping_perception.cpp

```diff
@@ -214,16 +216,17 @@ class BasicGraspingPerception : public rclcpp::Node
     auto result = std::make_shared<FindGraspableObjectsAction::Result>();
 
     // Get objects
-    find_objects_ = true;
+    need_to_find_objects_ = true;
     rclcpp::Time t = clock_->now();
-    while (find_objects_ == true)
+    while (need_to_find_objects_ == true)
+      // Keep looping to wait until another thread finds graspable objects.
```

Comment:
can we put the comment before the "while" - I just never like separating the while from the opening brace by a comment